src/modules/game.js

```diff
@@ -40,9 +43,14 @@ class Game {
     }
 
     start() {
-        this.castle.setup(this);
+        this.castle.setup(this, 3);
         this.answerForm.addEventListener('submit', this.handleAnswerSubmit);
         this.gameState = GAMESTATE.RUNNING;
+        // hide start page
+        startPage.style.display = 'none';
+        startPage.style.zIndex = -1;
+        gameOverPage.style.display = 'none';
+        gamePage.style.display = 'flex';
```

Comment:
Would be better if this was broken up into hideElement(element) and showElement(element, displayValue) functions, that could be in a different file, for example domUtils.